dc-infrastructure.tf

```diff
@@ -8,7 +8,7 @@ module "base-infrastructure" {
   desired_capacity = var.desired_capacity
   domain           = var.domain
   namespace        = local.namespace
-  share_home_size  = "5Gi"
+  share_home_size  = local.share_home_size
```

Comment:
/nit I know it is not part of this PR but technically, we should be using `shared_home_size` (instead of `share_home_*` prefix.

Reply:
done